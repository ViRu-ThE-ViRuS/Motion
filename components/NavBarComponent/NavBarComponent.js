import React, { Component } from 'react';

import {
    View,
    TouchableOpacity,
    StyleSheet,
} from 'react-native';

import { Icon } from 'react-native-elements';

export default class NavBarComponent extends Component {
    render() {
        return (
          <TouchableOpacity>
            <View style={styles.container}>
              <Icon
                name="search"
                type="ionicons"
                size={25} />
            </View>
          </TouchableOpacity>
        );
    }
}

const styles = StyleSheet.create({
    container: {
        marginRight: 15,
    },
});
