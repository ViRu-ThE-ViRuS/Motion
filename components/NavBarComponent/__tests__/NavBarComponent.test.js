import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import NavBarComponent from './../NavBarComponent';

describe('LoginButton tests', () => {
  it('FacebookLoginButton renders correctly', () => {
    const tree = renderer.create(
      <NavBarComponent />
    );

    expect(tree).toMatchSnapshot();
  });
});
