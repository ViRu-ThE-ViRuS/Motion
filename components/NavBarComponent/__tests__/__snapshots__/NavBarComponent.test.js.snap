// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButton tests FacebookLoginButton renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Object {
        "marginRight": 15,
      }
    }
  >
    <View
      onPress={undefined}
      style={
        Array [
          false,
          false,
          false,
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "justifyContent": "center",
          },
          undefined,
        ]
      }
      underlayColor="white"
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "black",
              "fontSize": 25,
            },
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ],
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        î¢¶
      </Text>
    </View>
  </View>
</View>
`;
